src/modules/catFacts/catFacts.schema.ts

```diff
@@ -6,9 +6,14 @@ export const catFactResponseSchema = z.object({
 })
 
 export const catFactParamsSchema = z.object({
-  id: z.preprocess(val => parseInt(val as string, 10), z.number({
-    invalid_type_error: "ID parameter must be a number"
-  }).int())
+  id: z.preprocess(
+    val => parseInt(val as string, 10),
+    z
```

Comment:
Thanks prettier, I hate it

Reply:
Yeah, I hate it too. Not sure what we can change in Prettier to make it better, if anything :/

Reply:
The aim of the game is uniformity to make diffs easier to read, if a dev needs/wants to format differently they can do that and lint-staged will fix it.

Reply:
* break it again